src/main/java/com/example/bookstoreapp/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package com.example.bookstoreapp.service.impl;
+
+import com.example.bookstoreapp.entity.Book;
+import com.example.bookstoreapp.repository.BookRepository;
+import com.example.bookstoreapp.service.BookService;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+
+    private final BookRepository bookRepository;
+
+    public BookServiceImpl(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
```

Comment:
Let’s  use RequiredArgsConstructor annotation

Reply:
I did it in Entity class, it's not enough?